src/java.base/windows/native/libjava/IOUtil.c

```diff
@@ -34,7 +34,7 @@
 #include "java_lang_Long.h"
 #include "nio.h"
 #include "nio_util.h"
-#include "net_util.h"
+/* #include "net_util.h" */
```

Comment:
Is this change intended or is this a left over from some experimentation?

Reply:
It could be leftover. There was a lot of wrangling in this. Probably line 37 could just be deleted.

Reply:
Line 37 deleted in f57b6f1.